pegasus/sites.v3/code.org/public/about/index.md

```diff
@@ -9,34 +10,50 @@ nav: about_nav
 
 # About Us
 
+[col-50]
+
+<% facebook = {:u=>'https://www.youtube.com/watch?t=66&v=mTGSiB4kB18'} %>
+<% twitter = {:url=>'https://www.youtube.com/watch?t=66&v=mTGSiB4kB18', :related=>'codeorg', :text=>'Anybody can learn computer science @codeorg #CSforAll'} %>
+
+<%=view :display_video_thumbnail, id: "codeorg_recruiting", video_code: "mTGSiB4kB18", play_button: 'center', facebook: facebook, twitter: twitter, letterbox: "false" %>
+
+[/col-50]
+
+[col-50]
+
+<div style="margin-left: 20px;">
+
 Launched in 2013, Code.org&reg; is a non-profit dedicated to expanding access to computer science, and increasing participation by women and underrepresented students of color. Our vision is that every student in every school should have the opportunity to learn computer science. We believe computer science should be part of core curriculum, alongside other courses such as biology, chemistry or algebra.
 
+</div>
+
+[/col-50]
+
+<div style="clear: both;"></div>
+
 ## Diversity of students in our courses
 ![image](/images/infographics/fit-800/diversity-courses.png)
 
 Code.org increases diversity in computer science by reaching students of all backgrounds where they are — at their skill-level, in their schools, and in ways that inspire them to keep learning. Read  about [our efforts to increase diversity in computer science](/diversity).
 
 <br />
 
-[col-50]
+<div style="float: left; width: 50%; padding: 10px;">
 
-**Code.org's work covered by CBS This Morning:**
+<%=view :display_video_thumbnail, id: "codeorg_cbs", video_code: "sUXfjzzHO5g", caption: "Code.org's work covered by CBS This Morning", play_button: "center", letterbox: "false" %>
```

Comment:
That still works for the URL (in dashboard & pegasus) to force it.  You need to provide a mp4 file though.